src/js/components/balances.js

```diff
@@ -1149,36 +1166,49 @@ function SweepModalViewModel() {
     multiAPIConsensus("create_issuance", transferData,
       function(unsignedTxHex, numTotalEndpoints, numConsensusEndpoints) {
         
-        var signedHex = key.checkAndSignRawTransaction(unsignedTxHex, [self.destAddress()]);
-        WALLET.broadcastSignedTx(signedHex, function(issuanceTxHash, endpoint) { //broadcast was successful
-          opsComplete.push({
-            'type': 'transferOwnership',
-            'result': true,
-            'asset': selectedAsset.ASSET,
-            'from': self.addressForPrivateKey(),
-            'to': self.destAddress()
-          });
-          PENDING_ACTION_FEED.add(issuanceTxHash, "issuances", transferData);
+        key.checkAndSignRawTransaction(unsignedTxHex, [self.destAddress()], function(err, signedHex) {
+          if (err) {
+            // @TODO: is this the correct way of handling err?
```

Comment:
@robby-dermody @ouziel-slama I can't really seem to understand the flow here, doing `callback(true, {` is correct here?

previously, incase it would fail it would just be uncaught ...

tbh I don't think it would make sense to retry if the signing fails ...